src/minimum_tracer/minimum_tracer.cpp

```diff
@@ -0,0 +1,3214 @@
+    }
+    else
+    {
+      if (tmp_T_high_newglob > T_low + 1)
+        tmp_T_high_newglob -= 1; // try to move out of problematic region
+    }
+  }
+  return;
+}
+
+void Vacuum::PrintPhasesDiagram(int size)
+{
+  std::stringstream ss;
+  std::vector<double> T_list;
+  std::vector<std::vector<double>> PhaseListPlot;
+  std::vector<AsciiPlotter> PlotList;
+  char markers[] = {".*ox-cs"};
+
+  for (std::size_t it = 0; it < modelPointer->get_nVEV(); it++)
+    PlotList.push_back(
+        AsciiPlotter(modelPointer->addLegendVEV()[it], size, ceil(size / 3.)));
+
+  for (std::size_t it = 0; it < PhasesList.size(); it++)
+  {
+    int nBefore =
+        floor(size * (PhasesList[it].T_low - T_low) / (T_high - T_low));
+    int nAfter =
+        floor(size * (T_high - PhasesList[it].T_high) / (T_high - T_low));
+    int nPhase = size - nBefore - nAfter;
+
+    // Calculate VEV positions
+    T_list.clear();
+    PhaseListPlot.clear();
+    for (int i = 0; i <= nPhase; i++)
+    {
+      T_list.push_back(PhasesList[it].T_low +
+                       (i / double(nPhase)) *
+                           (PhasesList[it].T_high - PhasesList[it].T_low));
+      PhaseListPlot.push_back(PhasesList[it].Get(T_list.back()).point);
+    }
+
+    // Calculate transpose
+    std::vector<std::vector<double>> PhaseListPlot_transpose(
+        PhaseListPlot[0].size(), std::vector<double>(PhaseListPlot.size()));
+    for (size_t i = 0; i < PhaseListPlot.size(); ++i)
+      for (size_t j = 0; j < PhaseListPlot[0].size(); ++j)
+        PhaseListPlot_transpose[j][i] = PhaseListPlot[i][j];
+
+    // Do the plotting
+    for (size_t i = 0; i < modelPointer->get_nVEV(); i++)
+      PlotList[i].addPlot(T_list,
+                          PhaseListPlot_transpose[i],
+                          "Phase " + std::to_string(it),
+                          markers[it % sizeof(markers)]);
+
+    /*ss << "Phase " << std::setw(3) << it << " |" << std::string(nBefore, ' ')
+       << std::string(nPhase, '.') << std::string(nAfter, ' ') << "|"
+       << std::setw(7) << PhasesList[it].T_low << " - " << std::setw(7)
+       << PhasesList[it].T_high << "|\n";*/
+
+    // Print also starting and ending minima
+    ss << "Phase " << std::setw(3) << it << " |" << std::string(nBefore, ' ')
+       << std::string(nPhase, '.') << std::string(nAfter, ' ') << "|"
+       << std::setw(7) << PhasesList[it].T_low << " - " << std::setw(7)
+       << PhasesList[it].T_high << "|"
+       << std::setw(2 * modelPointer->get_nVEV()) << std::setprecision(3)
+       << PhasesList[it].MinimumPhaseVector.front().point << "|"
+       << std::setw(2 * modelPointer->get_nVEV())
+       << PhasesList[it].MinimumPhaseVector.back().point << "\n";
+  }
+
+  for (std::size_t it = 0; it < modelPointer->get_nVEV(); it++)
+  {
+    PlotList[it].legend();
+    PlotList[it].xlabel("T (GeV)");
+    PlotList[it].ylabel("Phase");
+    PlotList[it].show(ss);
+  }
+
+  /*// Plotting smallest eigenvalue, used for debugging
```

Comment:
If you want this for debugging, you could wrap it in a extra flag or use a debug define. Or do you need it only sometimes during debugging?

Reply:
It is probably not needed anymore. I removed it.

Reply:
It is still here